charts/ever-gauzy-api/examples/demo-env-deployment-with-ingress.yaml

```diff
@@ -16,14 +15,14 @@ ingress:
     kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
   hosts:
-    - host: apidemo-charts.gauzy.co
+    - host: demo-api-gauzy.gauzy.co
       paths:
         - path: /
           pathType: Prefix
-  tls:
-    - secretName: chart-example-tls
-      hosts:
-        - apidemo-charts.gauzy.co
+  # tls:
```

Comment:
@Saif-Uz-Zaman should you uncomment that back? I need to add DNS for it??

Reply:
No, you don't need to add a DNS record for it. It's an example value. Clients need to change this value as per their environment.